src/initializers/kafka/kafka.ts

```diff
@@ -25,8 +25,21 @@ export default class Kafka {
     logger.info(`Kafka connected ${brokers.join(',')}`);
   }
 
-  public async send(topic: string, message: string | Buffer, key?: string, partition?: number) {
-    const { key: _key, partition: _partition, offset } = await this.producer.send(topic, message, partition, key);
+  public async send(
+    topic: string,
+    message: string | Buffer,
+    key?: string,
+    partition?: number,
+    headers?: { [key: string]: string }[]
+  ) {
+    const { key: _key, partition: _partition, offset } = await this.producer.send(
+      topic,
+      message,
+      partition,
+      key,
+      null,
+      headers
```

Comment:
Proposal: We could also add the producer name as a default header. WDYT?

Reply:
+1 as this way we could filter on the client messages coming from different environments.

Reply:
I'm not sure about the filtering:
1. What about ruby apps?
2. What about node apps that are using older orka?

Reply:
I will not be adding anything else in this PR. @klesgidis please open an enhancement request so we can discuss this.